tests/test_gtfs_objects.py

```diff
@@ -0,0 +1,251 @@
+
+    #routes.route_id_df
+    assert not routes.route_id_df.empty
+    assert len(routes.route_id_df) == len(routes.routes_df)
+    pd.testing.assert_frame_equal(routes.routes_df[[Route.ROUTES_COLUMN_ROUTE_ID_NUM, Route.ROUTES_COLUMN_ROUTE_ID]].set_index(Route.ROUTES_COLUMN_ROUTE_ID_NUM),
+                                  routes.route_id_df.set_index(Route.ROUTES_COLUMN_ROUTE_ID_NUM))
+
+    #routes.agencies_df
+    assert not routes.agencies_df.empty
+    #The SIMPLE network doesn't really do much with agency.
+    #TODO: Consider adding here once these tests includes more networks
+
+    #routes.fare_rules_df
+    assert not routes.fare_rules_df.empty
+
+
+    #routes.fare_attrs_df
+    assert not routes.fare_attrs_df.empty
+    fare_attrs_df_dtype = {
+        'fare_period': object,
+        'price': np.float64,
+        'currency_type': object,
+        'payment_method': np.int64,
+        'transfers': np.int64,
+        'transfer_duration': np.float64,
+    }
+    assert (set(fare_attrs_df_dtype.keys()).issubset(routes.fare_attrs_df))
+    assert routes.fare_attrs_df.dtypes.to_dict() == fare_attrs_df_dtype
+    assert len(routes.fare_attrs_df) == 9
+    match_fare_rule_attr_df = routes.fare_rules_df[routes.fare_rules_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD].isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]
+    assert len(match_fare_rule_attr_df) == len(routes.fare_rules_df)
+    #pd.testing.assert_frame_equal(routes.fare_rules_df[fare_attrs_df_dtype.keys()].drop_duplicates().set_index(Route.FARE_ATTR_COLUMN_FARE_PERIOD),
+    #                              routes.fare_attrs_df[routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD].isin(match_fare_rule_attr_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])].set_index(Route.FARE_ATTR_COLUMN_FARE_PERIOD),
+    #                              check_dtype=False, check_index_type=False)
+
+    #routes.fare_ids_df
+    assert not routes.fare_ids_df.empty
+    pd.testing.assert_frame_equal(routes.fare_rules_df[[Route.FARE_RULES_COLUMN_FARE_ID, Route.FARE_RULES_COLUMN_FARE_ID_NUM]].drop_duplicates().set_index(Route.FARE_RULES_COLUMN_FARE_ID_NUM),
+                                  routes.fare_ids_df.set_index(Route.FARE_RULES_COLUMN_FARE_ID_NUM))
+
+    #routes.fare_by_class
+    assert routes.fare_by_class
+
+    #routes.fare_transfer_rules_df
+    assert not routes.fare_transfer_rules_df.empty
+    assert len(routes.fare_transfer_rules_df[routes.fare_transfer_rules_df[Route.FARE_TRANSFER_RULES_COLUMN_FROM_FARE_PERIOD]
+        .isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]) == len(routes.fare_transfer_rules_df)
+    assert len(
+        routes.fare_transfer_rules_df[routes.fare_transfer_rules_df[Route.FARE_TRANSFER_RULES_COLUMN_TO_FARE_PERIOD]
+        .isin(routes.fare_attrs_df[Route.FARE_ATTR_COLUMN_FARE_PERIOD])]) == len(routes.fare_transfer_rules_df)
+
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_transfers_load(network_dir, gtfs_feed):
```

Comment:
add description

Reply:
Cleaned up test_gtfs_objects to be compatible with Partridge. Added docstrings.